README.md

```diff
@@ -6,5 +6,12 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
-1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
-1. Describe the flow of a PATCH request as it goes through the app.
+
+Rails exists as a tool for web developers to use to create and structure a back-end for their server.
+
+2. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what
+responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
Almost. The controller doesn't pas the request on to the model - it tries to carry out the request, and does so by calling the requisite model method(s). It then takes the output from that method and uses it to construct the response.